src/graphql/products/addToSellingPlan.ts

```diff
@@ -0,0 +1,52 @@
+// TODO: Debug this
+
+export default `
+query sellingPlanGroups {
+  sellingPlanGroup(id: "gid://shopify/SellingPlanGroup/1543929913") {
```

Comment:
You sure the correct syntax is with ""? I anyhow bombing

Reply:
yes its correct the query is running successfully with no errors